Detectors/TPC/workflow/include/TPCWorkflow/TPCFourierTransformAggregatorSpec.h

```diff
@@ -109,20 +107,22 @@ class TPCFourierTransformAggregatorSpec : public o2::framework::Task
   o2::ccdb::CcdbApi mDBapi;                     ///< API for storing the IDCs in the CCDB
   std::map<std::string, std::string> mMetadata; ///< meta data of the stored object in CCDB
   bool mWriteToDB{};                            ///< flag if writing to CCDB will be done
-  std::array<uint32_t, 2> mTFRange{};           ///< storing of first and last TF used when setting the validity of the objects when writing to CCDB
+  std::array<uint64_t, 2> mTimeStampRange{};    ///< storing of first and last time stamp used when setting the validity of the objects when writing to CCDB
   int mProcessedTFs{0};                         ///< number of processed time frames to keep track of when the writing to CCDB will be done
 
   /// \return returns TF of current processed data
   uint32_t getCurrentTF(o2::framework::ProcessingContext& pc) const { return o2::framework::DataRefUtils::getHeader<o2::header::DataHeader*>(pc.inputs().getFirstValid(true))->tfCounter; }
 
+  uint64_t getCurrentTimeStamp(o2::framework::ProcessingContext& pc) const { return DataRefUtils::getHeader<DataProcessingHeader*>(pc.inputs().getFirstValid(true))->creation; }
```

Comment:
Perhaps these two functions can be put in https://github.com/AliceO2Group/AliceO2/blob/dev/Detectors/TPC/workflow/include/TPCWorkflow/ProcessingHelpers.h
And I would prefer to rename `getCurrentTimeStamp` to `getCreationTime`.